src/Test.UnitTests.Sarif/Writers/PrereleaseCompatibilityTransformerTests.cs

```diff
@@ -142,5 +141,17 @@ public void PrereleaseCompatibilityTransformer_ThreadFlowLocationWithKind()
         {
             RunTest("ThreadFlowLocationWithKind.sarif");
         }
+
+        [Fact]
+        public void PrereleaseCompatibilityTransformer_FromSarifV1()
+        {
+            // We only need one test for the SARIF v1 conversion because for this
+            // transformation, the PrereleaseCompatibilityTransformer invokes the
+            // SarifVersionOneToCurrentVistor, which already has an extensive set
```

Comment:
>SarifVersionOneToCurrentVistor [](start = 15, length = 30)

nit: typo in the word Visitor

Reply:
Will fix and merge, thanks!

---
In reply to: [295439620](https://github.com/microsoft/sarif-sdk/pull/1538#discussion_r295439620) [](ancestors = 295439620)